tests/testthat/test-net-object.R

```diff
@@ -182,3 +182,24 @@ test_that("$set works as expected", {
   testObj$set(propName, as.integer(2))
   expect_equal(testObj$get(propName), 2)
 })
+
+test_that("$get works as expected for static fields", {
+  testObj <- newObjectFromName(rSharpEnv$testObjectTypeName)
+  fieldName <- "StaticString"
+
+  expect_equal(testObj$get(fieldName), "A static string")
+})
+
+test_that("$set works as expected for static fields", {
```

Comment:
Here a new test. I would expect it to work.

Reply:
Not sure I like this feature anymore :)

Is this commonly done in R programming? With this feature, especially the `set`, it's not clear to the R developer that he's changed all instances of this class system-wide.

This is not an allowed way to access static methods in C++/C# FWIW.

Reply:
Okay and what would be the proper syntax in C# for setting static fields?

Reply:
You access using the class name, not through the object. Which is supported using the r_call_static because you give that function a type instead of instance.